.github/workflows/no-response.yml

```diff
@@ -0,0 +1,20 @@
+name: No Response
+
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # Schedule for five minutes after the hour, every hour
+    - cron: '5 * * * *'
```

Comment:
I'm thinking about this just now after having approved the previous 2 PRs - does this need to happen every hour? Or is it enough if it happens once a day?

Reply:
Hmm yeah good point. Maybe once a day is enough! I'll update it in all of them.

Reply:
Changed!